bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/provider/AbstractResourceBundleProvider.java

```diff
@@ -406,6 +399,7 @@ protected Bundle getBundle(String uid) {
                     uri = new URI(prefix + ":" + uid + ".name");
                 } catch (URISyntaxException e) {
                     logger.error("Constructed invalid uri '{}:{}.name'", prefix, uid, e);
+                    return Collections.emptyList();
```

Comment:
We should return the default values here. I fixed it in https://github.com/openhab/openhab-core/pull/1161/files#diff-d90b601b2b68bbe8017ba644c98c6da1R387.